src/redux/actions/adminFormActions.js

```diff
@@ -0,0 +1,19 @@
+export const ACTION_TYPES = {
+  SET_COHORT_NAME: "SET_COHORT_NAME",
+  SET_COHORT_TYPE: "SET_COHORT_TYPE"
+}
+
+//Action creators
+export const setCohortName = cohortName => {
```

Comment:
I would add more comments on each action as well... just adding more comments on code.. make it more readable

Reply:
I've added the comments.